src/dotnet/Mgmt.CI.BuildTools/NugetToolsPackage/CI.Tools.Package/build/psModules/CodeGenerationModules/AutoRestCodeGenerationModule/AutoRestCodeGenerationModule.psm1

```diff
@@ -648,6 +648,9 @@ function Start-CodeGeneration {
     try {
         Install-AutoRest $AutoRestVersion
         Write-InfoLog "Commencing code generation"  
+
+        # Hard code to avoid x-ms-identifiers issue
+        $AutoRestVersion = "2.0.4421"
```

Comment:
This change seems to make the AutoRestVersion variable no longer configurable. Considering the status of track1 and the small possiblity of the necessity of using an v2 version, I'm approving the PR for a quick fix.